src/main/webapp/assets/modules/common/autoSuggestion.js

```diff
@@ -0,0 +1,20 @@
+techlooper.directive('autoSuggestion', function ($http) {
+  return {
+    restrict: "A",
+    scope: {
+      text: "=",
+      items: "=",
+      getUrl: "@"
+    },
+    link: function (scope, element, attr, ctrl) {
```

Comment:
'ctrl' is defined but never used.<br>'attr' is defined but never used.<br>'element' is defined but never used.